clouddriver-ecs/src/test/groovy/com/netflix/spinnaker/clouddriver/ecs/view/UnvalidatedDockerImageProviderSpec.groovy

```diff
@@ -0,0 +1,34 @@
+package com.netflix.spinnaker.clouddriver.ecs.view
+
+import com.netflix.spinnaker.clouddriver.ecs.provider.view.UnvalidatedDockerImageProvider
+import org.junit.Test
+import spock.lang.Specification
+
+class UnvalidatedDockerImageProviderSpec extends Specification {
+
+  @Test
+  void shouldNotHandleEcrRepositoryUrl() {
+    given:
+    UnvalidatedDockerImageProvider unvalidatedDockerImageProvider = new UnvalidatedDockerImageProvider()
+    String ecrRepositoryUrl = "769716316905.dkr.ecr.us-west-2.amazonaws.com/continuous-delivery:latest"
```

Comment:
Same here. Even if it's not real, I'd just do something _obviously_ fake like `111111111111`